dune-release fails with curl: command not found

`dune-release` fails with the following command:
```
[-] Creating release 4.3.0 on https://github.com/lemaetech/http-cookie.git via github's API
dune-release: [ERROR] curl execution failed: Non 0 exit code Exit code 127
  {status=Exit code 127; stderr="/bin/sh: line 1: curl: command not found
  ";
  stdout=""}
```

Curl is in the path
```
$ /bin/sh
sh-5.1$ curl --version
curl 7.86.0 (x86_64-pc-linux-gnu) libcurl/7.86.0 OpenSSL/3.0.7 zlib/1.2.13 brotli/1.0.9 zstd/1.5.2 libidn2/2.3.2 libssh2/1.10.0
 nghttp2/1.49.0
Release-Date: 2022-10-26
Protocols: dict file ftp ftps gopher gophers http https imap imaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tft
p 
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB SPNEGO SSL thre
adsafe TLS-SRP UnixSockets zstd
```

--- a/dune-project
+++ b/dune-project
@@ -25,7 +25,7 @@ with [Dune](https://github.com/ocaml/dune) and hosted on
 [GitHub](https://github.com).")
  (depends
   (ocaml (>= 4.08.0))
-  curly
+  (curly (>= 0.3.0))
   (fmt (>= 0.8.7))
   (fpath (>= 0.7.3))
   bos

--- a/dune-release.opam
+++ b/dune-release.opam
@@ -20,7 +20,7 @@ bug-reports: "https://github.com/tarides/dune-release/issues"
 depends: [
   "dune" {>= "2.7"}
   "ocaml" {>= "4.08.0"}
-  "curly"
+  "curly" {>= "0.3.0"}
   "fmt" {>= "0.8.7"}
   "fpath" {>= "0.7.3"}
   "bos"

